WordPress/WordPressTest/LikeUserHelperTests.swift

```diff
@@ -30,7 +36,6 @@ class LikeUserHelperTests: XCTestCase {
 
     func testNewLikeUserWithPreferredBlog() {
         let completionExpectation = expectation(description: "We expect the context to save successfully")
-        let contextManager = TestContextManager()
```

Comment:
As a rule IMHO we should aim to move _more_ things into the test body and out of the `setUp`, but this feels like a pragmatic alignment with how all the other tests work.

I believe we previously discussed something like this, which could be nice (though a `XCTest` subclass focused on Core Data could also save a ton of boilerplate – all things for future PRs!)

```swift
 withMockContextManager { context in 
 ...
}
```

Reply:
Having discovered [an unexpected behaviour](https://github.com/wordpress-mobile/WordPress-iOS/pull/18527#discussion_r867342970)(at least to me) of `XCTestCase` instance isn't released after test run. I don't think my change for this file is a reasonable change. Good point on creating a `XCTestCase` subclass, not sure why I didn't think of that 🤦‍♂️. Rather than reverting this one change for this particular test case, I'm thinking maybe I can create a follow up PR to create this subclass and use it in all test cases which need a CoreDataStack. What do you think?

Reply:
>  I can create a follow up PR to create this subclass and use it in all test cases which need a CoreDataStack. What do you think?

This works for me!